OSDOCS#11056: Agent docs cleanup (#78326)
openshift/openshift-docs: installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

* xref:../../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc#configuring-the-agent-based-installer-to-use-mirrored-images[Configuring the Agent-based Installer to use mirrored images]

[id="installing-ocp-agent-opt-manifests_{context}"]
=== Creating additional manifest files

> You can create additional manifests to further configure your cluster beyond the configurations available in the `install-config.yaml` and `agent-config.yaml` files.
> This is optional.

Reviewer: Or:

```suggestion
As an optional task, you can create additional manifests to further configure your cluster beyond the configurations available in the `install-config.yaml` and `agent-config.yaml` files.
```

Author: I do think this is a better option, thanks. It makes it clear from the beginning that this is optional, rather than tacking on the statement at the very end. I'll apply this change to the other procedures too.
